### DISCUSSION TITLE
Why is "colour.plotting.plot_RGB_colourspaces_gamuts" definition producing unexpected results?

### DISCUSSION THREAD

**Discussion by hminle** [colour-science/colour#750] [Q&A] [answered]
Hi Colour Team,

I use this snippet code to generate the gamuts in CIE XYZ:
![image](https://user-images.githubusercontent.com/16201681/90152794-38e9e500-dd56-11ea-8ecb-a1bac0bb2522.png)

![image](https://user-images.githubusercontent.com/16201681/90152705-1a83e980-dd56-11ea-8309-ddf89f9335e5.png)

My Colour version: ```0.3.15```

I think this result is wrong, because sRGB gamut lies outside ProPhoto RGB one.
Expected result:

![image](https://user-images.githubusercontent.com/16201681/90152954-66369300-dd56-11ea-8f4a-ede3338f3f2b.png)

sRGB gamut should lie entirely inside ProPhoto RGB one.

Could you help me check this out?
Did I write any thing wrong in my code?

Thanks

**Comment by thomasmansencal** [comment, marked as answer]
Hi @hminle,

This is a pure side effect of the illuminants of the two RGB colourspaces being different, i.e. D65 vs D50. What you can do is something like that:

```python
PPRGB_D65 = colour.models.RGB_COLOURSPACES['ProPhoto RGB'].chromatically_adapt(
    colour.ILLUMINANTS['CIE 1931 2 Degree Standard Observer']['D65'])

colour.plotting.plot_RGB_colourspaces_gamuts(
    colourspaces=[PPRGB_D65, 'sRGB'], reference_colourspace='CIE XYZ')
```

If you have a doubt like that, try **CIE xyY** and you will see that **ProPhoto RGB** encompasses almost entirely **sRGB** but they diverge at the top of the volume because of their different whitepoint.

You can also verify that here: https://www.colour-science.org:8020/

**Comment by hminle** [comment]
Hi @KelSolaar , 
Thank you
I thought D65 is the default white point for ProPhoto RGB.
So I use ProPhoto RGB colourspaces like this:
```python
prophotorgb_colourspace = colour.models.PROPHOTO_RGB_COLOURSPACE
```

In fact, I need to specify D65 clearly, right? Like what you did with `PPRGB_D65`

**Comment by hminle** [comment]
@KelSolaar 
Then how about `colour.plotting.plot_RGB_scatter`? I need to create PPRGB_D65 like that too?

**Comment by thomasmansencal** [comment]
> In fact, I need to specify D65 clearly, right? Like what you did with `PPRGB_D65`

Yes, if you really want to compare them apple to apple, they ideally all need to be chromatically adapted to the same whitepoint. Likewise for `colour.plotting.plot_RGB_scatter`. That discussion makes me think that we should maybe add that as an option as it could be a desirable default.

**Comment by hminle** [comment]
> > In fact, I need to specify D65 clearly, right? Like what you did with `PPRGB_D65`
> 
> Yes, if you really want to compare them apple to apple, they ideally all need to be chromatically adapted to the same whitepoint. Likewise for `colour.plotting.plot_RGB_scatter`. That discussion makes me think that we should maybe add that as an option as it could be a desirable default.

Yeah, I want to compare them like what you show me in the web: https://www.colour-science.org:8020/
I think many will want to do that.

**Comment by hminle** [comment]
I think if the parameter `colourspaces` is a list, then the function just needs to make sure all of colourspaces have the same white point

**Comment by thomasmansencal** [comment]
> I think if the parameter colourspaces is a list, then the function just needs to make sure all of colourspaces have the same white point

Yes and no because often we also want to compare, the colourspaces as they are defined by their primaries:

![image](https://user-images.githubusercontent.com/99779/90322567-f6204b00-dfa9-11ea-909a-6748a6b1c3bc.png)

I will certainly add an option to chromatically adapt them though!

**Comment by thomasmansencal** [comment]
@hminle : I have added support in the develop branch to chromatically adapt the RGB colourspaces in the various plotting definitions:

![image](https://user-images.githubusercontent.com/99779/90947467-3e4bdb80-e48a-11ea-84e0-16d1f5967add.png)
![image](https://user-images.githubusercontent.com/99779/90947477-528fd880-e48a-11ea-8f93-3f3bcbac4c4a.png)

**Comment by hminle** [comment]
Oh thank you a lot @KelSolaar
